contrib/endpoints/src/api_manager/service_control/aggregated.cc

```diff
@@ -85,6 +88,25 @@ CheckAggregationOptions GetCheckAggregationOptions(
                                  kCheckAggregationExpirationMs);
 }
 
+// TODO:: jaebong - need to add quota configuration
+// Generate QuotaAggregationOptions
+QuotaAggregationOptions GetQuotaAggregationOptions(
+    const ServerConfig* server_config) {
+  if (server_config && server_config->has_service_control_config() &&
+      server_config->service_control_config().has_check_aggregator_config()) {
```

Comment:
In the server_config proto file:

https://github.com/istio/proxy/blob/master/contrib/endpoints/src/api_manager/proto/server_config.proto

you may want to add QuotaAggregatorConfig, just like CheckAggregatorConfig